src/rest-api/trainer/spec/trainerApi.spec.ts

```diff
@@ -63,4 +65,34 @@ describe('TrainerApi', () => {
       });
     }).bind(this));
   });
+
+  describe('getExerciseEvaluationById', () => {
+    it('returns expected Exercise Evaluation by id equals 1', sinon.test((done) => {
+      // Arrange
+      const sinon: sinon.SinonStatic = this;
+
```

Comment:
The linter rule we are using forbids adding more than two consecutive blank lines, in this case I would put together all the const statements